charts/app/values.yaml

```diff
@@ -14,8 +14,8 @@ global:
   secrets:
     enabled: true
     databasePassword: ~
```

Comment:
Correct me if I'm wrong, this should be 'admin' too.

Reply:
It's supposed to autogenerate a password if none are provided. Did you not care and want that hard coded?

Reply:
I've only tried commenting 🤣 , I have no clue with how the backend was made 🤪